### PR TITLE
Quality of service patches

### PR DESCRIPTION
**Pull request by MaisenbacherD**
Linking the RocksDB repo in the `README.md`.
Also fixing the output for `zenfs --version`.

Signed-off-by: Dennis Maisenbacher <dennis.maisenbacher@wdc.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ZenFS: RocksDB Storage Backend for ZNS SSDs and SMR HDDs
 
-ZenFS is a file system plugin that utilizes RockDB's FileSystem interface to
+ZenFS is a file system plugin that utilizes [RockDB](https://github.com/facebook/rocksdb)'s FileSystem interface to
```
Comment (yhr) [inline README.md:3]: Typo.. RockDB --> RocksDB

**Review by yhr — CHANGES_REQUESTED** [review]
One small thing, otherwise this looks god

File: README.md (hunk 1 above)
Comment (MaisenbacherD) [reply README.md]: Fixed the typo :)

**Review by yhr — APPROVED** [review]

**Comment by yhr** [comment]
pushed manually

**Comment by MaisenbacherD** [comment]
The second commit (4af9bb5c1e4d7443fb7feafa2adfd67088bbee27) was not merged :)

**Comment by yhr** [comment]
fixed!

**Pull request closed without merging**